mgmtsystem/ec2.py

```diff
@@ -482,6 +482,11 @@ def upload_file_to_s3_bucket(self, bucket_name, file_path, file_name):
             self.logger.error("Error: File to upload does not exist.")
             return False
 
+    def object_exists_in_bucket(self, bucket_name, object_key):
+        bucket = self.s3_connection.Bucket(name=bucket_name)
+        objects = [o for o in bucket.objects.all() if o.key == object_key]
+        return any(objects)
```

Comment:
sorry for that mistake, in the modern api https://boto3.readthedocs.io/en/latest/reference/services/s3.html#S3.Client.head_bucket can be used instead